containers/test-apps/kitchen/bin/kitchen

```diff
@@ -599,6 +607,17 @@ class Kitchen(HTTPWebSocketsHandler):
     def __process_headers(self):
         """Handle logic for all supported Kitchen HTTP header values."""
         assert self.__path is not None, 'Processes headers AFTER processing the request path.'
+        global _default_response_status
+
+        # Temporarily set status on responses
+        default_status = self.headers.get('x-kitchen-default-status-value')
+        default_timeout = self.headers.get('x-kitchen-default-status-timeout')
```

Comment:
As discussed offline, let's make processing these headers optional, only enabled when a command-line flag is set. Maybe `--enable-status-change`?

Reply:
Done.